package.json

```diff
@@ -33,7 +33,8 @@
     "build-esm": "BABEL_ENV=esm babel src --out-dir esm --ignore '**/stories','**/test' --source-maps inline",
     "build": "concurrently --names 'commonjs,esm' 'yarn run build-commonjs' 'yarn run build-esm'",
     "clean": "git clean -Xdf",
-    "release": "np --no-publish",
+    "release": "np",
+    "build-docs": "cd docs && yarn install && yarn deploy",
```

Comment:
Should it be `deploy-docs` or `release-docs` (consistent with `release`) instead? Asking since it says `build`, but it actually deploys them.

Reply:
Ah, good call. That's much better!